contribution_guideline.md

```diff
@@ -0,0 +1,73 @@
+# Contributing to [ALX-DarkMode-project]
+
+Thank you for considering contributing to [ALX-DarkMode-project]! This file outlines the steps you should take to make a contribution to the project.
+
+## Getting Started
+
+Before you start contributing to the project, please make sure you have:
+- A GitHub account
+- Git installed on your local machine
+
+## Making Contributions
+
+Here are the general steps to follow when making contributions:
+
+1. **Fork the [ALX-DarkMode-project] repository on GitHub**.
+2. Clone your fork of the repository onto your local machine
+3. Create a new branch for your contribution: `git checkout -b my-branch`.
+4. Make changes to the code
+5. Test your changes to make sure they work as intended
+
+To view your changes on the intranet as you make your contribution, follow the guidelines below:
+
+- Open your browser and go to the ALX intranet.
+
+- Click on the extension icon on the top right part of your screen. It will display all the extensions you have downloaded on your system. If you don't have the ALX intranet theme selector extension, visit the Chrome extension market, download it, and switch to dark mode by clicking the extension and toggling the **ON** button.
+
+> **Note:** The following steps require using the ALX intranet theme selector extension.
+
+- Click on the extension icon again and select the three dots on the ALX intranet theme selector extension. This will reveal additional actions when you hover over it.
+- Choose "Manage extensions" from the available options.
+- Enable developer mode by clicking the toggle switch at the top right part of your screen. This shows three buttons: **Load unpacked**, **Pack extension**, and **Update**.
+- Click on **Load unpacked**.
+- Select the cloned ALX-DarkMode-project repository.
+- A popup will appear at the bottom left part of your screen showing "Extension loading". If this does not happen, repeat the process and ensure you have selected the correct repository.
+- Click the back arrow icon close to the ALX intranet theme selector. This will show all the extensions you have on your system, including two theme selectors: one sourced from the Chrome Web Store and the other sourced from your local machine (the cloned version).
+- Click on the **Details** button on the extension and scroll down to check the source of the extension. Also, take note of the reload icon on the extension.
+
+As you now have two extensions working on your system, it is advisable to turn off the extension from the Chrome Web Store and keep the local one enabled to easily track changes in the dark mode.
+
+- Click on the **OFF** button on the extension to turn it OFF.
+- Refresh your browser; you will notice that dark mode has been disabled because you turned it OFF (the one from the Chrome Web Store).
+- Click on the Chrome extension icon again.
+- Pin the ALX intranet theme selector for easy access.
```

Comment:
Kindly mention that the newly added extension (the one loaded locally) is what you're referring to here